src/commands/config/create.ts

```diff
@@ -0,0 +1,180 @@
+  revocation?: { type: string };
+  name: string;
+  documentStore?: string;
+  identityProof: {
+    type: string;
+    location?: string;
+    key?: string;
+  };
+}
+interface TransferableRecordIssuers {
+  name: string;
+  tokenRegistry: string;
+  identityProof: {
+    type: string;
+    location?: string;
+  };
+}
+
+const { trace } = getLogger("config:create");
+
+export const command = "create [options]";
+
+export const describe = "Create a config file";
+
+export const builder = (yargs: Argv): Argv =>
+  yargs
+    .option("output-dir", {
+      alias: "od",
+      description: "Write output to a directory",
+      type: "string",
+      demandOption: true,
+    })
+    .option("encrypted-wallet-path", {
+      type: "string",
+      description: "Path to file containing wallet.json",
+      normalize: true,
+    })
+    .option("config-template-path", {
+      type: "string",
+      description: "Path to file containing config template",
+      normalize: true,
+      demandOption: true,
+    });
+
+export const handler = async (args: CreateConfigCommand): Promise<void> => {
+  trace(`Args: ${JSON.stringify(args, null, 2)}`);
+  if (!fs.existsSync(args.outputDir)) {
+    fs.mkdirSync(args.outputDir);
+  }
+
+  try {
+    let walletPath = "";
+    if (!args.encryptedWalletPath) {
+      info(`Please enter password to create wallet`);
+      const createWalletParams = {
+        outputFile: `${args.outputDir}/wallet.json`,
+        fund: "ropsten",
+      };
+      walletPath = await CreatedWalletPath(createWalletParams);
+      info(`Wallet created at ${walletPath}`);
+    }
+    const walletFilePath = walletPath ? walletPath : args.encryptedWalletPath;
+    const wallet = await readFile(walletFilePath);
+    const walletObject = JSON.parse(wallet);
+
+    const configFile: ConfigFile = ConfigTemplate;
+    configFile.wallet = wallet;
+
+    const template = JSON.parse(await readFile(args.configTemplatePath));
+    const formsInTemplate = template.forms;
+    let documentStoreAddress = "";
+    let tokenRegistryAddress = "";
+    let verifiableDocumentDnsName = "";
+    let tokenRegistryDnsName = "";
+    const typeOfDocuments = formsInTemplate.map((formTemplate: any) => formTemplate.type);
+    if (typeOfDocuments.includes("VERIFIABLE_DOCUMENT")) {
+      info(`Enter password to continue deployment of Document Store`);
+      const deployDocumentStoreParams = {
+        encryptedWalletPath: walletFilePath,
+        network: "ropsten",
+        gasPriceScale: 1,
+        dryRun: false,
+        storeName: "Document Store",
+      };
+      const documentStore = await deployDocumentStore(deployDocumentStoreParams);
+      documentStoreAddress = documentStore.contractAddress;
+      success(`Document store deployed, address: ${highlight(documentStoreAddress)}`);
+      const DocumentStoreTemporaryDnsParams = {
+        networkId: 3,
+        address: documentStoreAddress,
+        sandboxEndpoint: "https://sandbox.openattestation.com",
```

Comment:
let's change the create dns implementation so that there is one place only where this is defined

Reply:
ok fixed.

Reply:
not done

Reply:
is it u mean that we defined it one place and replace the values to suit our needs?